docs/getting-started/create-host.md

```diff
@@ -158,11 +159,11 @@ const renderItem: RenderItemFunction = (item, index, level) => {
         return null;
     }
 
-    const { label, linkProps, additionalProps } = item;
+    const { label, to, linkProps, additionalProps } = item;
 
     return (
         <li key={`${level}-${index}`}>
-            <Link {...linkProps} {...additionalProps}>
+            <Link to={isDynamicTo(to) ? to() : to} {...linkProps} {...additionalProps}>
```

Comment:
Review note for myself. Validate to() here. Should there be a parameters passed to the func?